File: Data Structures & Algorithms/two-integer-sum/submission-2.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int n = nums.size();
        vector<int> arr(n);
        for(int i = 0; i < n; i++){
            arr[i]=target - nums[i];
            for(int j = i+1; j < n; j++){
                if (arr[i] == nums[j]){
                    return {i,j};
                }
            }
        }
        
    }
};

File: Data Structures & Algorithms/majority-element/submission-2.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        hashmap = {}
        n = len(nums)

        for num in nums:
            hashmap[num] = hashmap.get(num, 0) + 1
            if hashmap[num] > n//2:
                return num    